portfolio/src/main/webapp/index.html

```diff
@@ -12,11 +12,11 @@
         <h1>Welcome to my website!</h1>
         <hr>
         <h5>Where would you like to go?</h5>
-        <ul>
-            <li> About </li>
-            <li> Achievements </li>
-            <li> Contact </li>
-        </ul>
+        <div id="links">
+            <a href="about.html"> <p> About </p> </a>
+            <a href="achievement.html"> <p> Achievements </p> </a>
+            <a href="#"> <p> Contact </p> </a>
+        </div>
```

Comment:
I still think this should be an unordered list. Is there a reason you don't want to?